lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -144,6 +178,32 @@ class AwsCompileEventBridgeEvents {
               );
             }
 
+            if (RetryPolicy) {
+              if (!shouldUseCloudFormation) {
+                throw new ServerlessError(
+                  'Configuring RetryPolicy is not supported for EventBrigde integration backed by Custom Resources. Please use `provider.eventBridge.useCloudFormation` setting to use native CloudFormation support for EventBridge.',
```

Comment:
Inside error string, let's use `"` instead of backticks - it's a little bit more explicit in terminal output

Reply:
@pgrzesik prettier doesn't let me do that 😕

Reply:
also, just for the record, this line is heavily inspired by the `throw new ServerlessError(…, 'ERROR_INVALID_REFERENCE_TO_EVENT_BUS_CUSTOM_RESOURCE')` that was already in the code here.

Reply:
What is `prettier` complaining about here? 

@jpbochi Yeah, I know it's there in the other message, but it should not be with backticks there as well - it was a mistake. In a lot of other places we use `"` for error messages.

Reply:
Prettier automatically formats the file on save and replace the double quotes with simple quotes. When I disable the format on save setting I'm not able to commit due to the validation hook. 

<img width="684" alt="Screenshot 2021-09-08 at 11 37 44" src="https://user-images.githubusercontent.com/87330794/132530188-b4238c21-da54-4133-8d00-2c27c601c5b0.png">

Then I used a `--no-verify` flag and had no problem, but then I realised this is a terrible idea 'cause from now on every commit will have to use this flag.
Then I tried to ignore the quotes rule by adding  a `// eslint-disable-next-line quotes` before the lines we want to use double quotes, but then I'm not able to commit again 😕

 Do you have any other ideia to bypass this?

Reply:
This is quite surprising - do you have any additional prettier config interfering with the one of the Framework? We already have such cases of `"` inside strings e.g.: https://github.com/serverless/serverless/blob/df5f96952fad4f0593cb8b57b6a3ced2da7939e9/lib/plugins/aws/package/compile/functions.js#L168-L173 or https://github.com/serverless/serverless/blob/df5f96952fad4f0593cb8b57b6a3ced2da7939e9/lib/plugins/aws/provider.js#L2054-L2057

Reply:
Ahhh, there's a difference here: The double quotes are inside the template string. Let me try this way

Reply:
I reset my prettier config and we're good now. It was probably something interfering, as you said!